src/redux/projects/projects-thunk-creator.js

```diff
@@ -0,0 +1,20 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { getProjects } from './projects-reducer';
+import { projectsAPI } from '../../api/projectsApi';
+
+export const getProjectsThunk = createAsyncThunk(
+  'projects/getprojects',
```

Comment:
The REST route should look like GET `example.com/api/porjects`. Consider HTTP verb when writing your APIs.